### ISSUE TITLE
Accessibility icons

### ISSUE THREAD

**Issue by spydon** [ubuntu/yaru_icons.dart#193]
Hey hey!
I'm currently implementing the accessibility page in the Ubuntu installer and we're missing the pointing icon and the hearing icon, these ones:
```
/usr/share/icons/Yaru/scalable/categories/pointing-symbolic.svg
/usr/share/icons/Yaru/scalable/categories/hearing-symbolic.svg
```

It looks like it is fairly trivial to add new icons, so I could open a PR with them if you want?

**Comment by Feichtmeier** [comment]
This repo is basically Paul's work but I think I can be sure to say yes please :) 
With review

**Comment by Jupi007** [comment]
@spydon So sorry I didn't get the email notification... :/
I just did a PR, I will do a release once merged so you can use this.

**Comment by spydon** [comment]
Thanks a lot @Jupi007!

**Comment by Jupi007** [comment]
@spydon I just published the `2.4` release :)
